### PR TITLE
issue/872 – Display visit date and time in the Visits tab in the affiliate area.

### PR DESCRIPTION
**Pull request by DrewAPicture**
Issue: 872

### CODE REVIEW

File: templates/dashboard-tab-visits.php [hunk 1]
```diff
@@ -45,6 +46,9 @@
 								<i></i>
 							</span>
 						</td>
+						<td>
+							<?php echo esc_html( mysql2date( 'Y-m-d H:i:s', $visit->date ) ); ?>
```
Comment (pippinsplugins) [inline templates/dashboard-tab-visits.php]: Shouldn't this use date_i18n() instead?

Comment (pippinsplugins) [inline templates/dashboard-tab-visits.php]: Also probably best to set the format with the `date_format` option.

Comment (DrewAPicture) [inline templates/dashboard-tab-visits.php]: @pippinsplugins I thought about that, except we also want to show then time, at which point we're getting into *two* options pulls and logic to display it, which I'm not sure necessarily belongs in a template. So we can create a helper or something, but in this instance, I used the same hard-coded date format used in the list table.

Comment (pippinsplugins) [inline templates/dashboard-tab-visits.php]: Fair point. Let's leave as is for now then.

Comment (DrewAPicture) [inline templates/dashboard-tab-visits.php]: @pippinsplugins In response to the question about using `date_i18n()`, we should probably pass `true` as the third argument to `mysql2date()` to ensure the time string is passed through `date_i18n()` and therefore translated. The reason `date_i18n()` isn't used here directly, is that it expects a unix timestamp whereas `mysql2date()` expects a date string which is then converted into a unix timestamp.

**Pull request merged**